com.adobe.Flash-Player-Projector.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "com.adobe.Flash-Player-Projector",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flashplayer",
+    "separate-locales": false,
+    "tags": ["proprietary"],
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Play sounds */
+        "--socket=pulseaudio"
+    ],
+    "modules" : [
+        {
+            "name" : "flashplayer",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "install -D -t /app/bin/ apply_extra",
+                "install flashplayer /app/bin",
+                "install -Dm644 -t /app/share/icons/ com.adobe.Flash-Player-Projector.svg",
```

Comment:
still the wrong path: `/app/share/icons/hicolor/scalable`

Reply:
FWIW, the path that's currently there works just as well. Fixed.